components/camera/server.go

```diff
@@ -93,7 +93,7 @@ func (s *serviceServer) GetImage(
 		// network. Just to be discarded.
 		img, release, err = castedCam.Read(ctx)
 	default:
-		img, release, err = ReadImage(gostream.WithMIMETypeHint(ctx, req.MimeType), cam)
+		img, release, err = cam.GetImage(gostream.WithMIMETypeHint(ctx, req.MimeType))
```

Comment:
In which scenarios will we actually hit the default path here instead of going through ReadImager?

Reply:
I think builtins? @seanavery

Reply:
But I think since everything is being united under `GetImage` or soon `Image` we _can_ delete the `Read` path